### DISCUSSION TITLE
Only edited files? (Git)

### DISCUSSION THREAD

**Discussion by kentlarsson** [streetsidesoftware/vscode-spell-checker#2370] [Q&A]
Would it be possible to configure the plugin to only check files i've edited? Or maybe even parts of a file I've edited?

I'm working in a big codebase and it unfortunately gets a bit spammy. At the same time I think the extension is awesome and would like to use it to fix some spelling errors and not introduce new ones, baby steps.

**Comment by Jason3S** [comment]
I have added this as an issue: #2371.

At the moment, the extension only checks files that are open.
